NUService.js

```diff
@@ -156,6 +156,22 @@ export default class NUService extends NUObject {
         return url;
     }
 
+    buildBulkURL(entities, RESTResourceName, parentEntity) {
+        if (!entities || !entities[0]) {
+            return null;
+        }
+
+        let url = `${this.rootURL}/`;
+        url += ((!parentEntity) ? '' : `${parentEntity.resourceName}/${parentEntity.ID}/`);
+        url += `${entities[0].data.resourceName}?responseChoice=1`
+        entities.forEach(({data}) => 
+            {
+                url += `&id=${data.ID}`
```

Comment:
We are sending the entities in the request body, are we required to append the IDs at the end of the URL like this?

Reply:
Server code is expecting ids in the url. not the request body